core/src/parameter.jl

```diff
@@ -74,13 +74,18 @@ end
 Base.to_index(id::NodeID) = Int(id.value)
 
 const ScalarInterpolation = LinearInterpolation{
-    ReadOnlyVector{Float64, Vector{Float64}},
-    ReadOnlyVector{Float64, Vector{Float64}},
+    Vector{Float64},
+    Vector{Float64},
     Vector{Float64},
     Vector{Float64},
     Float64,
 }
 
+set_zero!(v) = fill!(v, zero(eltype(v)))
+const LBC = LazyBufferCache{Returns{Int}, typeof(set_zero!)}
```

Comment:
May I suggest renaming `LBC` to `Cache` for readability? With a docstring explaining what we cache.